docs/source/procedures/datadog/cloud-integration.rst

```diff
@@ -0,0 +1,60 @@
+=============================================
+ScyllaDB Cloud Monitoring Datadog Integration
+=============================================
+
+For security reasons, the ScyllaDB cloud does not have direct access to the Prometheus server.
+To allow external server scrapping, you will need to enable the Prometheus proxy.
+The Datadog agent reads from the proxy, which reads from the Promethues server.
+
+1. Installing and configuring the Datadog Agent.
+2. Add Datadog recording rules.
+3. Loading Scylla dashboard to Datadog.
+4. Optionally load Monitor (Alerts).
+
+Scylla Monitoring Datadog Integration Overview
+==============================================
+A typical ScyllaDB cluster generates thousands of metrics, sometimes even tens of thousands.
+The sheer number of metrics is too much for Datadog.
+
+Instead of letting the Datadog agent scrap all metrics, the monitoring stack marks a small subset of metrics with a label and lets the Datadog agent scrap only those.
+
+Install And configure the Datadog Agent
+=======================================
+
+Start by following `Installation <https://docs.datadoghq.com/agent/>`_ guide. The datadog agent should run on a machine that can reach the Prometheus Proxy server.
```

Comment:
Its not clear whre to install the agent.
Maybe something like:
Start by installing DataDog Agent on a server with access to ScyllaDB Cloud Prometheus Proxy server...

A small digram would help.

Reply:
Link to https://cloud.docs.scylladb.com/stable/monitoring/cloud-prom-proxy.html